Source/WTF/Scripts/GeneratePreferences.rb

```diff
@@ -208,6 +208,11 @@ def experimental?
   def internal?
     %w{ unstable internal }.include? @status
   end
+  
+  # Features which should be enabled while running tests
+  def testable?
+    %w{ testable preview }.include? @status
```

Comment:
`stable` should be treated  like `preview`. There's not much difference between the two. 

(Also this is what is causing the GTK failures)

Reply:
No -- stable should be kept in its default state. If GTK relied on something in the ExperimentalFeatures yaml file to be turned on for testing, it should use 'testable' state.

Reply:
I think this is problematic because you can't have per-platform statuses. Hence I think maybe the best solution is to add `stable` to that list. Either way, it doesn't make much sense to me to treat `preview` differently from `stable`, either both should be kept, or removed. `preview` is just another stable configuration.

Reply:
No -- `preview` indicates a feature that is not yet on by default in shipping Safari, but should be on in STP. This is a very important distinction.

For now, it seems like enabling `stable` features is this simplest action at present.

Likely Elliott and I will need to add a port-specific feature for state so we can have distinct settings for each port.